pkg/commonvpc/vpc.go

```diff
@@ -737,3 +733,11 @@ func getTableConnEffect(connQuery, conn *connection.Set) (*connection.Set, vpcmo
 		return conn.Intersect(connQuery), vpcmodel.PartlyAllow
 	}
 }
+
+// MultipleVPCsConfigPrefix returns the vpcName of the passed resource when config is multi-vpc
+func MultipleVPCsConfigPrefix(c *vpcmodel.VPCConfig, resource *vpcmodel.VPCResource) string {
+	if c != nil && resource.VPC() != nil {
```

Comment:
when is `resource.VPC()` expected to be `nil`?

Reply:
I got some tests failed before I added this condition, it seems like there are some cases we do not init config file in tests

Reply:
only unit tests or end-to-end tests?
this is not about initializing config file, but having the resource a pointer to the VPC it is contained in.